sample_grabber.c

```diff
@@ -116,7 +116,8 @@ static void print_usage(void)
   "                    Valid range 0x0001 to 0xFFFF.\n"
   "  [--help -h]     Print usage information and exit.\n"
   "  [--onebit -1]   Convert samples to packed 1-bit format (MSB first)\n"
-  "  [--rotate -r]   Rotate files hourly for long-term archive\n"
+  "  [--rotate -r INTERVAL]\n"
+  "                  Rotate files every INTERVAL seconds for long-term archive\n"
```

Comment:
good to mention INTERVAL but better to note that it's optional, defaults to hourly